page.codeberg.JakobDev.jdProcessFileWatcher.yaml

```diff
@@ -0,0 +1,72 @@
+id: page.codeberg.JakobDev.jdProcessFileWatcher
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: jdprocessfilewatcher
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_PYWEBENGINE=1
+finish-args:
+  - --filesystem=xdg-data/flatpak/exports/share/applications:ro
+  - --filesystem=/var/lib/flatpak/exports/share/applications:ro
```

Comment:
I wonder how that work if the path is non standard.

Reply:
No, but there's nothing I can do about it. Luckily this can only be [done by editing a config file](https://docs.flatpak.org/en/latest/tips-and-tricks.html#adding-a-custom-installation), so majority will use the standard,